.github/workflows/alphabetise.yml

```diff
@@ -0,0 +1,53 @@
+name: Alphabetise extension list
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - antora-playbook.yml
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20'
+
+      - name: Install packages
+        run: npm install -g yaml-sort
+
+      - name: Alphabetise
+        id: alpha
+        uses: mikefarah/yq@master
+        with:
+          path: antora-playbook.yml
+          cmd: '.content.sources |= sort_by(.url)'
```

Comment:
I **think** GitHub already comes with `yq` installed, so I don't think we need this action

Reply:
If not found, maybe use this action to install it, since we're already using it in our Quarkiverse extensions: https://github.com/quarkiverse/quarkus-ironjacamar/blob/main/.github/workflows/quarkus-snapshot.yaml#L37-L38

Reply:
`jq` for sure. `yq` I'm not sure at all.